### PR TITLE
update check on UUID branch names #2732

### PR DESCRIPTION
**Pull request by hchauvin**

### CODE REVIEW

File: src/client/pkg/uuid/uuid_test.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	// Hexadecimal
+	if IsUUIDWithoutDashes("09abcd098faa4fd98643023485739xyz") {
+		t.Fail()
+	}
+
+	// Generated
+	if !IsUUIDWithoutDashes(NewWithoutDashes()) {
+		t.Fail()
+	}
+}
```
Comment (jdoliner) [inline src/client/pkg/uuid/uuid_test.go]: Check out `src/client/pkg/require` it has some helper functions which would make these all 1 liners.
They'll also print out a bit more information when the test does fail.

**Comment by jdoliner** [comment]
Looks good to me, 1 comment about using `require` in the test to make things a bit shorter and more in line with our other tests.

**Comment by hchauvin** [comment]
done

**Comment by jdoliner** [comment]
Merging this despite the fact that CI is failing, the first CI is failing because of creds issues on external PRs. The second is failing because of a linter issue that has been fixed on master. The other parts of CI passed and it seems silly to block this until we shuffle around some commits to get the right thing to run when the meaningful checks have already run.

**Pull request merged**